mobile/src/common/enums/notification/notification.ts

```diff
@@ -1 +1,2 @@
-export { NotificationType } from './notification-type.enum';
+export { NotificationMessage } from './notification-messages/notification-message.enum';
+export { NotificationType } from './notification-type/notification-type.enum';
```

Comment:
We don't need to wrap each enum/type/constant in separate folder
```suggestion
export { NotificationMessage } from './notification-message.enum';
export { NotificationType } from './notification-type.enum';
```

Reply:
I will remember it